docs/validators/guides/block-production.md

```diff
@@ -0,0 +1,84 @@
+# Block production
+Koinos uses the novel consensus algorithm, Proof-of-Burn (PoB). To participate in block production you will need to configure your node as well as perform some simple actions on the blockchain.
+
+---
+## Proof-of-Burn basics
+Proof-of-Burn has similarities to both Proof-of-Work (PoW) and Proof-of-Stake (PoS) with some notable differences. Like Proof-of-Stake, no mining occurs to produce blocks; this places all physical nodes on an equal playing field. Hashing is simulated in Proof-of-Burn by burning KOIN in order to gain Virtual Hash Power (VHP). Aside from using VHP rather than physical machines performing hashes, the algorithm works similarly to Proof-of-Work.
+
+---
+## Prerequisites
+Please follow our guide on [running a node](running-a-node.md). Be sure to start the node with the `--profile all` or `--profile block_producer` option to start the optional block producer microservice.
+
+---
+## Retrieving your address and key
+Upon starting a new node, a private block production key will be automatically generated for you. This key will be at `$KOINOS_BASEDIR/block_producer/private.key` (`$KOINOS_BASEDIR` is `~/.koinos` on macOS/Linux and `C:\koinos` for Windows if following our guide). There is a corresponding `public.key` file that is written out when the block producer runs and is always the public key corresponding to the private key that the block producer is configured to use (`private.key` by default).
+
+You will want to copy the contents of `public.key` and save this for later use. This is our "hot" key.
+
+```console
+cat $KOINOS_BASEDIR/block_producer/public.key
+```
+```{ .txt, .no-copy }
+Aq4Ps_Ch-f8OZDnpQOov2SiMvdYyA5tn0oWa36QWnTeH
+```
+
+Next, you will want your main account to hold VHP and KOIN for block production. Let us use the `koinos-cli` to register the PoB contract and open the wallet for our main account.
+
+```{ .txt, .no-copy }
+koinos-cli -r http://localhost:8080/
+🔐 > register pob 159myq5YUhhoVWu3wsHKHiJYKPKGUrGiyv
+🔐 > open <wallet_file> <password>
+```
+
+Let us grab our main account address so that we can register it within the Proof-of-Burn contract.
+
+```{ .txt, .no-copy }
+🔓 > address
+Wallet address: 1P4msR22FXKHZragcLk2dCNweTEi9JWgxn
+```
+
+---
+## Registering your key and burning
+Using the address from our main account and the public key from our block producer we will now create the association in the Proof-of-Burn contract.
+
+```{ .txt, .no-copy }
+🔓 > pob.register_public_key 1P4msR22FXKHZragcLk2dCNweTEi9JWgxn Aq4Ps_Ch-f8OZDnpQOov2SiMvdYyA5tn0oWa36QWnTeH
+```
+
+Next, we must burn some KOIN in order to receive VHP. Let us burn 10,000 KOIN in exchange for 10,000 VHP. The `pob.burn` call is requesting that 10,000 KOIN be burned from the first address and place 10,000 VHP in to the second address. In our case, we want the address to burn KOIN from to be the same that receives VHP.
```

Comment:
Should we reduce this amount to be more accessible to readers? Something not as large as thousands of dollars?